sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Stddev.scala

```diff
@@ -79,3 +81,116 @@ case class StddevPop(
     }
   }
 }
+
+// Compute standard deviation based on online algorithm specified here:
+// http://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
+abstract class StddevAgg(child: Expression) extends DeclarativeAggregate {
```

Comment:
This is included for benchmark, could be used to replace the imperative one (should be done in follow up PR or this one?)